src/Lecture1.hs

```diff
@@ -119,4 +134,14 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Int -> [Int] -> [Char]
+lowerAndGreater n list = go 0 0 list
+    where
+        -- go :: Int -> Int -> [Int] -> [Char] 
```

Comment:
You can actually keep type signatures for functions inside `where` and I would strongly encourage you to do so! 👍🏻